.github/workflows/nightly-build.yml

```diff
@@ -136,6 +136,8 @@ jobs:
 
     - name: Setup Rust toolchain and cache
       uses: actions-rust-lang/setup-rust-toolchain@v1.5.0
+      with:
+        rustflags: ''
```

Comment:
Used single quotes to keep consistency. (Single quotes are mainly used inside this file)